doc/rst/language/spec/modules.rst

```diff
@@ -409,18 +515,9 @@ the following:
  * ``this`` to indicate the requested module is a submodule of the
    current module
 
-The names that are made visible by a ``use`` statement are inserted in to a new
-scope that immediately encloses the scope within which the statement
-appears. This implies that the position of the ``use`` statement within a scope
-has no effect on its behavior. If a scope includes multiple ``use`` statements
-or a combination of ``use`` and ``import`` statements, then the newly-visible
-names are inserted in to a common enclosing scope.
-
-An error is signaled if multiple conflicting enumeration constants or public
-module-level symbols would be inserted into this enclosing scope with the same
-name, and that name is accessed by other statements in the same scope as the
-use.  Remember that this does not apply to functions unless they are also
-indistinguishable in other ways, see :ref:`Function_Overloading`.
+An error is signaled if symbols brought in by this ``use`` are accessed and
+would conflict with other symbols, see :ref:`Use_And_Import_Conflicts` for more
+information.
```

Comment:
I don't think this needs to be here in this section at all but it is OK with me if it's important to you.